docs/src/content/docs/fr/resources/themes.mdx

```diff
@@ -67,6 +67,12 @@ Installez un thème créé par la communauté pour personnaliser rapidement l'ap
 			href: 'https://delucis.github.io/starlight-theme-flexoki/',
 			previews: { light: 'flexoki-light.png', dark: 'flexoki-dark.png' },
 		},
+		{
+			title: 'Starlight Nova',
+			description: 'Un superbe thème moderne pour Starlight.',
```

Comment:
Didn't go with "beau" or "joli" for "beautiful" because it felt a bit weird to me when reading it out loud, so I slightly tweaked the sentence to avoid it.